extensions/amp-auto-ads/0.1/amp-auto-ads.js

```diff
@@ -72,8 +72,8 @@ export class AmpAutoAds extends AMP.BaseElement {
     return xhrFor(this.win)
         .fetchJson(configUrl, xhrInit)
         .catch(reason => {
-          dev().error(TAG, 'amp-auto-ads config xhr failed: ' + reason);
-          return null;
+          user().error(TAG, 'amp-auto-ads config xhr failed: ' + reason);
+          throw reason;
```

Comment:
> This also cancels the entire promise queue if fetching/parsing of the configuration fails.

This is specifically to stop further processing, since we can't do anything without a config.